docs/preview.md

```diff
@@ -3,8 +3,9 @@
 Ruff includes an opt-in preview mode to provide an opportunity for community feedback and increase confidence that
 changes are a net-benefit before enabling them for everyone.
 
-Preview mode enables a collection of newer lint rules, fixes, and formatter style changes that are
-considered experimental or unstable,.
+Preview mode enables a collection of new lint rules, fixes, formatter style and other changes that are considered experimental or unstable.
+
+Simply enabling preview mode does not imply that all the new preview features/changes will be turned on. Some, like the rules that are in preview, need to be turned on explicitly (see the [rules section](#using-rules-that-are-in-preview). Others, like the formatter style or default values changes will be turned on by simply enabling the preview mode.
```

Comment:
I find this a little confusing. We have a flag that determines if preview rules require "explicit" selection but by default they do not i.e. "implicit" selection of the rule via a prefix is sufficient. I'd be much more direct here:

```suggestion
Enabling preview mode does not select all preview rules by default. See the [rules section](#using-rules-that-are-in-preview) for details on selecting preview rules.
```

I'd also consider putting it in a note admonition.